tests/fixtures/ember-container-query-glint/output/src/components/widgets/widget-1.ts

```diff
@@ -1,6 +1,7 @@
 import templateOnlyComponent from '@ember/component/template-only';
 
 interface WidgetsWidget1Signature {
+  Args: {};
```

Comment:
**TODO**

Ideally, the output files in `ember-container-query-glint` should match the input files exactly. See if we can later remove `Args` from `Signature` when the component does not receive arguments.